flag/service/namespaceauth/namespaceauth.go

```diff
@@ -1,5 +1,6 @@
 package namespaceauth
 
 type NamespaceAuth struct {
-	ViewAllTargetGroup string
+	ViewAllTargetGroup     string
+	TenantAdminTargetGroup string
```

Comment:
So tenant admin is a customer admin with access to the CP k8s, right?

Reply:
Right. It has access to create app/appcatalogs and cluster/machindeployments CRs